### PR TITLE
Minor changes

### PR DESCRIPTION
**Pull request by priyanka9634**
added brand-switcher in mobile view and added new info-cards

### CODE REVIEW

File: index.ftd [hunk 1]
```diff
@@ -1,6 +1,20 @@
 -- import: fifthtry.com/library as lib
 
 
+-- string left-content:
```
Comment (amitu) [inline index.ftd:4]: Always leave one empty line between the "section/subsection line", line that starts with `-- ` or `--- `, and the "body". What comes right after must be `key: value` headers/properties/attributes.

File: index.ftd [hunk 2]
```diff
@@ -1,6 +1,20 @@
...
+Our suite of products are open source and available on Github. You are free to download install and customize them to your needs.
```
Comment (amitu) [inline index.ftd:5]: Ensure lines are not too long, put a line break after 80 characters or so.

File: index.ftd [hunk 3]
```diff
@@ -1,6 +1,20 @@
...
+
+
+We’d love to hear your feedback and suggestions, and collectively make Documentation easier and better for everyone 
+
+
+
+-- string right-content:
```
Comment (amitu) [inline index.ftd:12]: Line gap here as well.

File: index.ftd (hunk 2 above)
Comment (gsalunke) [reply index.ftd]: this is being followed now!

File: index.ftd (hunk 3 above)
Comment (gsalunke) [reply index.ftd:18]: done!

File: library.ftd [hunk 4]
```diff
@@ -1456,6 +1456,139 @@ width: fill
...
+
+--- info-mobile:
+if:$config.is-mobile
+right-title: $right-title
+left-title: $left-title
+right-logo: $right-logo
+left-logo: $left-logo
+left-body: $left-body
+right-body: $right-body
+
```
Comment (gsalunke) [inline library.ftd:1497]: @priyanka9634 always add 10 line gap between 2 components.

File: library.ftd [hunk 5]
```diff
@@ -1456,6 +1456,139 @@ width: fill
...
+--- info-card: $right-title
+body: $right-body
+sites-logo: $right-logo
+
+
+
+
+
+
```
Comment (gsalunke) [inline library.ftd:1534]: @priyanka9634 only 10 lines gap!

File: library.ftd [hunk 6]
```diff
@@ -1456,6 +1456,139 @@ width: fill
...
+
+
```
Comment (gsalunke) [inline library.ftd:1584]: @priyanka9634 10 lines gap only!

File: library.ftd [hunk 7]
```diff
@@ -1741,12 +1874,12 @@ optional string link:
 fpm.toc-item list children:
 ftd.color text-color: $fpm.color.glass.text-strong
 
---- ftd.row:
+/--- ftd.row:
```
Comment (gsalunke) [inline library.ftd:1882]: @priyanka9634 remove commented! also why `border` is missing from `border-color: $fpm.color.main.` ?

File: library.ftd [hunk 8]
```diff
@@ -2290,3 +2423,36 @@ border-width: 0
...
+text: $body
+role: $fpm.type.copy-relaxed
+color: $fpm.color.main.text
+
+--- container: ftd.main
+
+--- ftd.image:
+src: $sites-logo
+width: 81
+height: auto
```
Comment (gsalunke) [inline library.ftd:2463]: @priyanka9634 add line after 2458

**Review by gsalunke — APPROVED** [review]

**Pull request merged**